amt/core/exception_handlers.py

```diff
@@ -5,23 +5,43 @@
 from fastapi.responses import HTMLResponse
 from starlette.exceptions import HTTPException as StarletteHTTPException
 
+from amt.api.deps import templates
+
 logger = logging.getLogger(__name__)
 
 
 async def http_exception_handler(request: Request, exc: StarletteHTTPException) -> HTMLResponse:
     logger.debug(f"http_exception_handler: {exc.status_code} - {exc.detail}")
-    return HTMLResponse(
+
+    if request.state.htmx:
```

Comment:
There is no test that hits this condition.

Reply:
solved